src/actions/auth/auth.js

```diff
@@ -1,100 +1,28 @@
-import { callV1Api } from '../../helpers/api';
-import { recordBooksList } from '../recordBooks';
+export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
+export const LOGOUT_SUCCESS = 'LOGOUT_SUCCESS';
+export const SET_CURRENT_USER = 'SET_CURRENT_USER';
 
-export const LOGIN_POST_ERROR = 'LOGIN_POST_ERROR';
-export const LOGIN_POST_REQUESTING = 'LOGIN_POST_REQUESTING';
-export const LOGIN_POST_SUCCESS = 'LOGIN_POST_SUCCESS';
-export const LOGIN_VALIDATE = 'LOGIN_VALIDATE';
-export const LOGOUT_DELETE_SUCCESS = 'LOGOUT_DELETE_SUCCESS';
+export const loginSuccess = (token, uid) => {
+  localStorage.setItem('X-USER-TOKEN', token);
+  localStorage.setItem('X-USER-UID', uid);
```

Comment:
Shouldn't this be done in the reducer?

Reply:
Probably XD

Reply:
Lol. Yeah. We should just be passing the data to the reducer and the reducer handles the rest.